src/index.jsx

```diff
@@ -1,9 +1,25 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { applyMiddleware, createStore, combineReducers } from 'redux';
+import thunk from 'redux-thunk';
+import { createLogger } from 'redux-logger';
+import { Provider } from 'react-redux';
 import { BrowserRouter as Router } from 'react-router-dom';
 import App from './components/App/App';
+import movieReducers from './reducers/movies';
+import singleMovieReducer from './reducers/singleMovie';
+import sortReducer from './reducers/sort';
 import './index.scss';
 
-const renderRoutes = () => <Router><App /></Router>;
+const logger = createLogger();
+const reducers = combineReducers({
+  movie: singleMovieReducer,
+  movies: movieReducers,
+  sortBy: sortReducer,
+});
+const middleware = applyMiddleware(logger, thunk);
+const store = createStore(reducers, window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__(), middleware); // eslint-disable-line
```

Comment:
Let's use redux-devtools-extention instead of window.__REDUX ... 
https://github.com/zalmoxisus/redux-devtools-extension#14-using-in-production